modules/workload-identity/variables.tf

```diff
@@ -85,7 +85,7 @@ variable "automount_service_account_token" {
 }
 
 variable "roles" {
-  description = "A list of roles to be added to the created service account"
+  description = "A list of roles to be added to the created service account for specific projects"
```

Comment:
Currently this would be a breaking change, if possible can we make this backward compatible.  

A quick thought might be add a new (optional) `map(list(string))` of additional projects=>[roles], and leave the existing `roles` for just `var.project_id`?

Reply:
Changes are made as mentioned. Pls review